packages/client/src/subscriptions/types/getSubscriptions.types.ts

```diff
@@ -0,0 +1,32 @@
+import type { Config } from '../../types';
+
+export type GetSubscriptions = (
+  query?: GetSubscriptionsQuery,
+  config?: Config,
+) => Promise<Subscription[]>;
+
+export type Subscription = {
+  id: string;
+  metadata?: unknown;
```

Comment:
Isn't metadata always an object or undefined? If that is the case, I would type it as an object | undefined instead of unknown.